e2e-test/src/test/features/BigQueryToDistinct.feature

```diff
@@ -0,0 +1,157 @@
+@Distinct
+Feature: Distinct analytics - Verify BigQuery source data transfer using Distinct Analytics
+
+  @BQ_SOURCE_DISTINCT_TEST @BQ_SINK_TEST @PLUGIN-1206
```

Comment:
Add @albertshau to take a look. As discussed offline, shouldn't we use the plugin in the test repo to create the pipeline. e.g: use `File` instead of `BigQuery` in `hydrator-plugin` repo.

Reply:
Changes done.